### PR TITLE
UHM-6181, attach the specimen collection encounter to the correct visit

### PR DESCRIPTION
**Pull request by cioan**
Hi @mogoodrich , these changes here are just for fixing the problem with attaching the specimen collection encounter to the correct visit and for making the Processed Date widget visible. I am going now through the list of other issues you found and documented thoroughly in the tickets. Thanks!

### CODE REVIEW

**Review by mogoodrich — APPROVED** [review]
LGTM, thanks @cioan .  Added one comment/suggestion but defer to you as to whether that would be helpful or not?

File: omod/src/main/webapp/resources/scripts/components/LabTrackingAddOrderController.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ angular.module("labTrackingAddOrderController", [])
                 else if ( $scope.visitStopDateTime && $scope.order.requestDate.value > $scope.visitStopDateTime ) {
                     $scope.order.requestDate.value = $scope.visitStopDateTime;
                 }
+                $scope.order.sampleDate.value = $scope.order.requestDate.value;
```
[the remark this answers is not in the document]
Comment (cioan) [reply omod/src/main/webapp/resources/scripts/components/LabTrackingAddOrderController.js]: Thanks @mogoodrich ! I have just made another commit where I removed the requestDate variable. Please let me know if I missed anything. Thanks!

**Review by mogoodrich — APPROVED** [review]
LGTM, thanks @cioan !

**Comment by cioan** [comment]
Thanks @mogoodrich !

**Pull request merged**